src/TimelineID/TimelineID.test.tsx

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import { TimelineId } from './TimelineId';
+import { MemoryRouter, Route } from 'react-router-dom'
+import { mockAllIsIntersecting } from 'react-intersection-observer/test-utils';
 
-
-test('renders Rover explorer link', () => {
-    const { getByText } = render(<TimelineId />);
-    const linkElement = getByText(/The current Timeline Id is:/i);
-    expect(linkElement).toBeInTheDocument();
+test('renders Rover Timeline', () => {
+    const queries = render(<MemoryRouter initialEntries={["/timeline/perserverance"]}>
+        
+            <Route path="/timeline/:roverName">
+                <TimelineId />
+            </Route>
+    </MemoryRouter>);
+    mockAllIsIntersecting(true);
```

Comment:
does this just make sure that everything is 'onScreen'?

Reply:
Yes I had to mock a test because something was clashing with the intersection scope property in react